code/modules/vending/vending.dm

```diff
@@ -617,6 +618,13 @@ ADMIN_INTERACT_PROCS(/obj/machinery/vending, proc/throw_item)
 							account["current_money"] -= product.product_cost
 						else
 							src.credit -= product.product_cost
+					if (!player_list)
+						wagesystem.shipping_budget += round(product.product_cost * profit) // cogwerks - maybe money shouldn't just vanish into the aether idk
+					else
+						//Players get 90% of profit from player vending machines QMs get 10%
+						var/obj/machinery/vending/player/T = src
```

Comment:
`T` as a var name is only acceptable for turfs`

Reply:
Oh sorry, I just moved the Topic code over and didn't think about variable names. Fixed now